doc/source/reference/c-api.ufunc.rst

```diff
@@ -114,8 +166,15 @@ Functions
     :param nout:
         The number of outputs
 
+    :param identity:
+
+        Either :c:data:`PyUFunc_One`, :c:data:`PyUFunc_Zero`,
+        :c:data:`PyUFunc_None`. This specifies what should be returned when
+        an empty array is passed to the reduce method of the ufunc.
+
     :param name:
-        The name for the ufunc.  Specifying a name of 'add' or
+        The name for the ufunc as a ``NULL`` terminated string.
+        Specifying a name of 'add' or
         'multiply' enables a special behavior for  integer-typed
```

Comment:
Might justify these lines together for readability here.

Reply:
Reflow would be nice.

Reply:
fixed